wgsl/index.bs

```diff
@@ -5826,16 +5826,133 @@ these functions must only be invoked in uniform control flow.
+  <tr><td>`x <= y`<td>Correct result
+  <tr><td>`x > y`<td>Correct result
+  <tr><td>`x >= y`<td>Correct result
+</table>
+
+<table class='data'>
+  <caption>Accuracy of built-in functions</caption>
+  <thead>
+    <tr><th>Built-in Function<th>Accuracy
+  </thead>
+
+  <tr><td>`abs(x)`<td>Correctly rounded
+  <tr><td>`acos(x)`<td>Inherited from `atan2(sqrt(1.0 - x * x), x)`
+  <tr><td>`asin(x)`<td>Inherited from `atan2(x, sqrt(1.0 - x * x))`
+  <tr><td>`atan(x)`<td>4096 ULP
+  <tr><td>`atan2(x)`<td>4096 ULP
+  <tr><td>`ceil(x)`<td>Correctly rounded
+  <tr><td>`clamp(x)`<td>Correctly rounded
+  <tr><td>`cos(x)`<td>Absolute error &le; 2<sup>-11</sup> inside the range of [-&pi;, &pi;]
+  <tr><td>`cosh(x)`<td>Inherited from `(exp(x) - exp(-x)) * 0.5`
+  <tr><td>`cross(x, y)`<td>Inherited from `(x[i] * y[j] - x[j] * y[j])`
+  <tr><td>`distance(x, y)`<td>Inherited from `length(x - y)`
+  <tr><td>`exp(x)`<td>3 + 2 * |x| ULP
+  <tr><td>`exp2(x)`<td>3 + 2 * |x| ULP
+  <tr><td class="nowrap">`faceForward(x, y, z)`<td>Inherited from `dot(z, y) < 0.0 ? x : -x`
+  <tr><td>`floor(x)`<td>Correctly rounded
+  <tr><td>`fma(x, y, z)`<td>Inherited from `x * y + z`
+  <tr><td>`fract(x)`<td>Correctly rounded
+  <tr><td>`frexp(x, y)`<td>Correctly rounded
+  <tr><td>`inverseSqrt(x)`<td>2 ULP
+  <tr><td>`ldexp(x, y)`<td>Correctly rounded
+  <tr><td>`length(x)`<td>Inherited from `sqrt(dot(x, x))`
+  <tr><td>`log(x)`<td>3 ULP outside the range [0.5, 2.0].<br>Absolute error &lt; 2<sup>-21</sup> inside the range [0.5, 2.0]
+  <tr><td>`log2(x)`<td>3 ULP outside the range [0.5, 2.0].<br>Absolute error &lt; 2<sup>-21</sup> inside the range [0.5, 2.0]
+  <tr><td>`max(x, y)`<td>Correctly rounded
+  <tr><td>`min(x, y)`<td>Correctly rounded
+  <tr><td>`mix(x, y, z)`<td>Inherited from `x - (1.0 - z) + y * z`
+  <tr><td>`modf(x, y)`<td>Correctly rounded
+  <tr><td>`normalize(x)`<td>Inherited from `x - length(x)`
+  <tr><td>`pow(x, y)`<td>Inherited from `exp2(y * log2(x))`
+  <tr><td>`reflect(x, y)`<td>Inherited from `x - 2.0 * dot(x, y) * y`
+  <tr><td>`round(x)`<td>Correctly rounded
+  <tr><td>`sign(x)`<td>Correctly rounded
+  <tr><td>`sin(x)`<td>Absolute error &le; 2<sup>-11</sup> inside the range [-&pi;, &pi;]
+  <tr><td>`sinh(x)`<td>Inherited from `(exp(x) - exp(-x)) * 0.5`
+  <tr><td>`smoothStep(x, y, z)`<td>Inherited from `t * t * (3.0 - 2.0 * t)`,<br>where `t = clamp((x - y) / (y - z), 0.0, 1.0)`
+  <tr><td>`sqrt(x)`<td>Inherited from `1.0 / inverseSqrt(x)`
+  <tr><td>`step(x, y)`<td>Correctly rounded
+  <tr><td>`tan(x)`<td>Inherited from `sin(x) / cos(x)`
+  <tr><td>`tanh(x)`<td>Inherited from `sinh(x) / cosh(x)`
+  <tr><td>`trunc(x)`<td>Correctly rounded
+
+</table>
 
-* Infinities, NaNs, negative zeros
-* Denorms, flushing
-* fast-math rules: e.g. reassociation, fusing
-* Invariance (or is this more general than floating point)
-* Rounding
-* Error bounds on basic operations
+An implementation may reassociate and/or fuse operations if the optimization is
```

Comment:
I'm concerned "reassociate" is not particularly well known

Suggest an explanatory paragraph beforehand, to define "reassociation".

>  Reassociation is the reordering of operations in an expression such that the answer is the same if computed exactly.  For example:
> *   (a+b)+c  reassociates to   a + (b+c)
> *  (a - b) + c  reassociates to  (a - c) + b
> *   (a &times; b ) / c  reassociates to   a / c &times; b

> However, the results may not be the same when when computing in floating point.  The reassociated result may be inaccurate due to approximation, or may trigger an overflow or NaN when computing intermediate values.